test/integration/apiserver/oidc/oidc_test.go

```diff
@@ -152,7 +152,7 @@ func runTests(t *testing.T, useAuthenticationConfig bool) {
 
 				if useAuthenticationConfig {
 					authenticationConfig := fmt.Sprintf(`
-apiVersion: apiserver.config.k8s.io/v1beta1
```

Comment:
Do we want to keep some testing for older versions too? looks like when we added v1beta1, we still had v1alpha1 as well? (eye-balling the rest of the changes in this file)

Reply:
I think just keeping the v1beta1 load tests is fine, the other tests can switch to v1

Reply:
done!